lib/logger.js

```diff
@@ -1,7 +1,57 @@
 'use babel'
 
+import _ from 'lodash'
+
 export default class Logger {
-  error (/* statusCode, result, builder */) {}
-  warning (/* message */) {}
-  info (/* message */) {}
+  constructor () {
+    this.messages = []
+    this._group = null
+    const callback = () => { this.showFilteredMessages() }
+    atom.config.onDidChange('latex.loggingLevel', callback)
```

Comment:
The more that I think about it, I am leaning towards removing the dynamic filtering since it's meanless for `DefaultLogger`. Thoughts?